tests/umock_c_int/test_dependency.h

```diff
@@ -21,6 +21,12 @@ extern "C" {
         int x;
     } TEST_STRUCT;
 
+    typedef struct TEST_NESTEED_STRUCT_TAG
```

Comment:
>STEED [](start = 26, length = 5)

I too am a noble steed!

Reply:
Fixed.

---
In reply to: [399036417](https://github.com/Azure/umock-c/pull/154#discussion_r399036417) [](ancestors = 399036417)